Frontend/src/components/header/header.css

```
@@ -105,10 +105,10 @@
 
 .nav-menus li a{
     text-decoration: none;
-    color: white;
+    color: #fff;
```

please use the variables in here
```suggestion
    color: var(--tertiary-color);
```